pacifica/metadata/orm/base.py

```diff
@@ -209,14 +209,18 @@ def _date_operator_compare(date, kwargs, dt_converts=datetime_converts):
     def where_clause(cls, kwargs):
         """PeeWee specific extension meant to be passed to a PeeWee get or select."""
         where_clause = Expression(1, OP.EQ, 1)
+        # if 'include_deleted' not in kwargs and 'deleted' not in kwargs:
```

Comment:
These lines should be removed.

Reply:
Ok, I've removed those lines, but Travis seems to be complaining about code that I didn't change (not to my knowledge, anyhow)

Reply:
Again, not sure why pylint keeps puking.